features/networking/multus.feature

```diff
@@ -416,4 +416,107 @@ Feature: Multus-CNI related scenarios
     Then the output should contain "net1"
     And the output should contain "net2"
     And the output should contain 2 times:
-      | macvlan mode bridge |
+	    | macvlan mode bridge |
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24488
+  @admin
+  Scenario: Create pod with Multus bridge CNI plugin without vlan
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
+    And the multus is enabled on the cluster
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project_name clipboard
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/bridge-host-local-novlan.yaml |
+      | n | <%= cb.project_name %>                                                                                               |
```

Comment:
`\` is in an newline

Reply:
Thanks @zhaozhanqi  Can you clarify more? Didn't understand

Reply:
oh, Nothing, should be the display issue in my side.

Reply:
thanks